### DISCUSSION TITLE
Portal Install

### DISCUSSION THREAD

**Discussion by groiso** [YetiForceCompany/YetiForceCRM#15820] [General]
Hi,

Is the API URL the address of the Yeti server?

What URL should I put in the Web Service - Application?

Thanks

**Comment by mariuszkrzaczkowski** [comment]
you have an example
![image](https://user-images.githubusercontent.com/10198654/160299615-47b9de6f-22ca-4cc3-8926-1e67f59ae46c.png)

**Reply by groiso** [reply]
Hi
OK.
My question is which API URL should I put?

Is this URL that appears default?

**Reply by mariuszkrzaczkowski** [reply]
CRM URL: https://gitdeveloper.yetiforce.com/

API URL: https://gitdeveloper.yetiforce.com/webservice/

**Reply by groiso** [reply]
None of the URLs work, I didn't leave the screen you showed above.
I'm trying to install CustomerPortal2 via standard webservice to Yetiforce 6.3.0.

**Reply by mariuszkrzaczkowski** [reply]
If it does not work, it means that there is something wrong with communication with CRM.
This requires debugging and full information on both environments

**Reply by groiso** [reply]
The installation is in a different domain, could this be it?

Why doesn't the installation start? everything is working fine

**Reply by groiso** [reply]
Is there any other configuration to be done that besides enabling Web Service,
-> type: Standard Webservice
-> Public URL: https://gitdeveloper.yetiforce.com/webservice/
-> Login and Password.
Generates the key to enter the Customer Portal.

**Reply by mariuszkrzaczkowski** [reply]
https://yetiforce.com/en/knowledge-base/documentation/implementer-documentation/item/how-to-install-yetiforce-portal

**Reply by groiso** [reply]
Hi, @mariuszkrzaczkowski 

I have already followed this tutorial, but the URL is not clear. When installing the Customer Portal, the URL is in default, but can be edited.
Do I have to leave the default URL? But still the screen doesn't even start the installation

**Reply by groiso** [reply]
![Screenshot_8](https://user-images.githubusercontent.com/95945251/160478057-1daa6af7-5035-478e-bfa7-4a7182080507.png)
The installation does not leave this point

**Reply by groiso** [reply]
Hi, @mariuszkrzaczkowski 

I found that the Hostgator server doesn't allow installation of Yarn in shared plan.

I appreciate the help.

Thanks

**Reply by mariuszkrzaczkowski** [reply]
you don't need yarn, attach full portal and crm logs

**Reply by groiso** [reply]
I have this error in the domain log

 session_name(): Cannot change session name when headers already sent in .../public_html/crm/app/Session/Base.php on line 48
[30-Mar-2022 11:15:03 America/New_York] PHP Warning:  session_set_cookie_params(): Cannot change session cookie parameters when headers already sent in .../public_html/crm/app/Session/Base.php on line 49
[30-Mar-2022 11:15:03 America/New_York] PHP Warning:  session_set_save_handler(): Cannot change save handler when headers already sent in .../public_html/crm/app/Session.php on line 37
[30-Mar-2022 11:15:03 America/New_York] PHP Warning:  session_start(): Cannot start session when headers already sent in .../public_html/crm/app/Session.php on line 39

**Reply by mariuszkrzaczkowski** [reply]
check carefully if your server meets the requirements, because it looks like it does not
https://yetiforce.com/pl/component/zoo/item/web-server-requirements

**Reply by groiso** [reply]
Hi, @mariuszkrzaczkowski 

After doing a new installation and reconfiguring the php.ini, as required by the system, is attached, I still have the same problem and installation error.


Sorry, the problem may even be basic, but the error is passing and I can't find the obvious.

obs. There are two domains, one with CRM and the other with the Portal

**Reply by mariuszkrzaczkowski** [reply]
Do CRM and portal have the same configuration? https://yetiforce.com/en/knowledge-base/documentation/administrator-documentation/item/logs-server-configuartion

Attach the CRM configuration report and full logs https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/debugging

**Reply by groiso** [reply]
Hi, @mariuszkrzaczkowski 

Hi,

Sorry for this inconvenience and thanks for helping.
I made a copy of the existing log and the .htaccess and ConfReport file.
The crm and portal are in different domains
The .htaccess file (1) is the one recommended by the YetiForce doc and the .htaccess file (2) is the file created during the installation.

Some settings I can not change, because the server in shared account, does not allow.

**Reply by mariuszkrzaczkowski** [reply]
You have a wrong configuration, the APi server will not work until you correct it

**Reply by groiso** [reply]
Hi, @mariuszkrzaczkowski 

I read many doc and tutorials. Can you send me a correct link of this tutorial please.

**Reply by mariuszkrzaczkowski** [reply]
You have everything described here: https://yetiforce.com/en/knowledge-base/documentation/implementer-documentation/item/web-server-requirements

**Reply by groiso** [reply]
Hi, @mariuszkrzaczkowski

Sorry, but I've read and reread this doc many times and I can't find the problem to work. It may even be something obvious and it's in my face, but I swear I can't find it.

It has nothing about API or portal

**Reply by mariuszkrzaczkowski** [reply]
Report to your administrator to correct the configuration

**Comment by groiso** [comment]
Hi, @mariuszkrzaczkowski 

The hostgator server does not allow some changes to php and/or MYSQL/MariaDB in shared account. I've tried to change it, but without success.
If I knew exactly what was going wrong I would fix it.
But I see that there is no way.

**Reply by mariuszkrzaczkowski** [reply]
So CRM will not work properly on this server

**Reply by groiso** [reply]
Hi, @mariuszkrzaczkowski 

Okay, thank you very much and sorry for the inconvenience.
I would like to adapt and/or create a module in Yeti Force for tourism, as I did with vTiger. I was thinking about migrating because of the Portal.
Thanks again.

**Reply by groiso** [reply]
Hi, @mariuszkrzaczkowski 

Does this doc https://doc.yetiforce.com/api/ change anything? I don't know anything here

**Reply by mariuszkrzaczkowski** [reply]
> Hi, @mariuszkrzaczkowski
> Does this doc https://doc.yetiforce.com/api/ change anything? I don't know anything here
Here is a description of how the API works, it will be useful when the API will work for you

**Comment by groiso** [comment]
Hi, @mariuszkrzaczkowski 

In the system.log file, this Portal message appears

2022-03-31 13:18:29 [warning] File not found: /home1/cctur254/public_html/portal/public_html/layouts/Default/modules/Base/resources/Install.js
#0 app/Controller/View.php:121  >>  App\Log->warning('File not found: public_html/layouts/Default/modules/Base/resources/Install.js')
#1 app/Controller/View.php:298  >>  App\Controller\View->convertScripts([array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array,array],'js')
#2 app/Controller/View.php:225  >>  App\Controller\View->getFooterScripts()
#3 modules/Install/View/Install.php:94  >>  App\Controller\View->postProcess(true)
#4 app/WebUI.php:132  >>  YF\Modules\Install\View\Install->postProcess()
#5 app/WebUI.php:73  >>  App\WebUI->triggerPostProcess()
#6 index.php:31  >>  App\WebUI->process()
REQUEST = Array
(
    [module] => Install
    [view] => Install
    [_csrf] => sid:d304532c537061bd9b6de3a37ba4565feed7d778,1648737857
    [mode] => step2
    [lang] => en-US
)

**Reply by mariuszkrzaczkowski** [reply]
You haven't read it carefully, everything has already been written what to do, what is needed for a full problem analysis, you don't
then no one will help you
